frontend/src/theme/buttonStyles.tsx

```diff
@@ -0,0 +1,43 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme";
+import textStyles from "./textStyles";
+
+const Button: ComponentStyleConfig = {
+  baseStyle: {
+    color: "white.default",
+    width: "auto",
+    ...textStyles["button-medium"],
+  },
+  variants: {
+    primary: {
+      background: "cyan.300",
+      transition: "background-color 0.25s ease !important",
+      _hover: {
+        background: "cyan.600",
+        transition: "transition: background-color 0.5s ease !important",
+      },
+      _active: {
+        background: "cyan.300",
+        boxShadow: "inset 0px 0px 10px #ffffff !important",
+      },
+    },
+    secondary: {
```

Comment:
Yeah that makes sense! I added the secondary button too. I'm trying to make it so that we can pass the color in as props from I'm having issues >:( its not letting me `import type { StyleFunctionProps } from '@chakra-ui/styled-system'`... if anyone knows anything about this that would be great, if not then tmr problem